applications/P2PStateChannels.md

```diff
@@ -0,0 +1,254 @@
+
+# Team members
+
+### Partially grant supported:
+
+Founder - Luka Stanisic
+https://github.com/lukastanisic99
+https://www.linkedin.com/in/luka-stanisic-712907251/
+
+Engineer - Dragan Basta
+https://github.com/thepeaknick
+https://www.linkedin.com/in/dragan-basta/
+
+Designer - Aleksa Krstic
+https://www.linkedin.com/in/aleksa-krstic-124880112/
+
+Business development - Aleksa Gradinski
+https://www.linkedin.com/in/aleksa-gradinski/
+
+Business development - Gerard Murphy
+https://www.linkedin.com/in/gerard-murphy-8939b520/
+
+### Advisors / Not grant supported:
+
+Cryptography - Jeffrey Burdges
+https://github.com/burdges
+https://www.linkedin.com/in/jeffreyburdges/
+
+Economics & legal - Kasper Mai Jørgensen
+https://www.linkedin.com/in/kmjoergensen/
+
+Engineering - Andrew Burger
+https://github.com/coax1d
+https://www.linkedin.com/in/andrew-b-8a628870
+
+
+# Contact
+
+- **Contact Name:** Luka Stanisic
+- **Contact Email:** lukastanisic2@gmail.com
+
+# Team's experience
+
+Luka - Background in engineering, protocol level of different DLTs (DAGs and blockchains), distributed systems at Microsoft Azure, Polkadot Blockchain Academy alumni with distinction.
+
+Dragan - Software engineer with 10+ years of experience.
+
+Aleksa Krstic - Senior UX/UI designer at another major DLT.
+
+Aleksa Gradinski - In crypto since 2017, CEO of Agroblock ($1M annual profit), helped in negotiations and expansion of leading web3 companies in Serbia.
+
+Gerard - Engineering degree holder with over a decade in high-end business development, overseeing multiple offices in the London property market. 15 years of professional poker playing and game theory expertise, bringing a strategic and analytical approach to business growth and a specific understanding of peer-to-peer commercial opportunities.
+
+Jeff - Applied cryptographer and mathematician at the Web3 Foundation. One of the lead protocol designers behind Polkadot and Kusama.
+
+Kasper - ex CFO Web3 Foundation and co-founder Polimec.
+
+Andrew - Researcher/Engineer at the Web3 Foundation
+
+## Development Status :open_book:
+
+In progress.
+
+## Development Roadmap :nut_and_bolt:
+
+Provide an SDK with a minimal set of features for alpha testing as soon as possible.
+
+# Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):** Currently 1 person 24/7 and many part time with the team target going forward being 7 FTE
+- **Total Costs:** $100000 - this covers the scope of this implementation
+- **DOT %:** 50% (we're open to taking 100% in DOT, with 50% being vested over 2 years)
+
+# Milestone 1 and budget — Minimal feature set (MFS)
+The minimal feature set (MFS) is the minimal product that's runnable and it includes:
+  - Permissionless p2p discovery and connection establishment through Holepunch,
+  - Opening the channel on-chain,
+  - Fully p2p state machine progression - running native smart contracts fully p2p.
```

Comment:
We'd some question about this having only one milestone, no?  As I understand it, this milestone covers much of what's layed out in the Project Details section above.

I've left comments above on norner cases, some of which I've discossed with Luka, and on extensions, which mostly I did not previously discuss with Luka.  I think none of those comments represent architectural weaknesses, just stuff one might change before doing some applicaiton divferent form the envisioned applciations.

It's fairly involved, and Luka & Dragan have a fair amount of work here, with some of it like holepunch requiring a bunch of debugging work.  At the same time, they have largely worked out the architectural challenges in their prototyping.  Also, holepunch is a framework that "eventually works".  So they're not really in need of a bunch of smaller milestones.